pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+**Name**
```

Comment:
good call 👍  but I would make this change in a separate PR - these changes are not related to current PR.

Reply:
Ok, up to you.